nexus/lib/machines.py

```diff
@@ -2999,7 +2999,7 @@ def post_process_job(self,job):
                 job.gpus = 6 # gpus to use per node
             #end if
             if job.alloc_flags is None:
-                job.alloc_flags = 'smt1'
+                job.alloc_flags = 'gpumps smt4'
```

Comment:
This changes Summit to 4 threads (from one) and enables the MPS server.  Is this an intended part of this PR?

Reply:
Not really... On my simulations I sometimes noticed a benefit in using 'gpumps smt4' rather than 'smt1',
but it is file to use the latter as default.
Honestly, I forgot to undo that change before doing the pull request.

Reply:
Reverted. We can reconsider when/if there is stronger data.